shared/constants/types/more.js

```diff
@@ -1,10 +1,20 @@
 // @flow
 
 import {Component} from 'react' // eslint-disable-line
-import type {Device as _Device} from './flow-types'
 
 export type DeviceType = 'mobile' | 'desktop' | 'backup'
-export type Device = {type: DeviceType} & _Device
+import type {Device as _Device, DeviceID, Time} from './flow-types'
+
+export type Device = {
+  name: string;
+  deviceID: DeviceID;
+  type: DeviceType;
+  created: Time;
+  currentDevice: boolean;
+  provisioner: ?_Device;
```

Comment:
This is used in the details page for a single device. it's used in the 'timeline' view